### PR TITLE
refactor(ekf_localizer): use covariance index util of tier4_autoware_util

### PR DESCRIPTION
**Pull request by scepter914**
Signed-off-by: scepter914 <scepter914@gmail.com>

## Description

Refactor covariance index in ekf_localizer.
Move covariance index to util library merged by #1840.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by kminoda — APPROVED** [review]
LGTM

File: localization/ekf_localizer/src/ekf_localizer.cpp [hunk 1]
```diff
@@ -648,10 +652,10 @@ void EKFLocalizer::publishEstimateResult()
   twist_cov.header.stamp = current_time;
   twist_cov.header.frame_id = current_ekf_twist_.header.frame_id;
   twist_cov.twist.twist = current_ekf_twist_.twist;
-  twist_cov.twist.covariance[0] = P(IDX::VX, IDX::VX);
-  twist_cov.twist.covariance[5] = P(IDX::VX, IDX::WZ);
-  twist_cov.twist.covariance[30] = P(IDX::WZ, IDX::VX);
-  twist_cov.twist.covariance[35] = P(IDX::WZ, IDX::WZ);
+  twist_cov.twist.covariance[COV_IDX::X_X] = P(IDX::VX, IDX::VX);
```
Comment (YamatoAndo) [inline localization/ekf_localizer/src/ekf_localizer.cpp:655]: @scepter914 This value is the Twist, not the Pose.
So,  please modify `COV_IDX::X_X` to `TWIST_COV_IDX::VX_VX`).

Comment (scepter914) [reply localization/ekf_localizer/src/ekf_localizer.cpp:655]: @YamatoAndo 
Thank you for review.
I modify covariance index name in https://github.com/autowarefoundation/autoware.universe/pull/1950.
So after merged this PR, I'll adjust the covariance index name.

Comment (scepter914) [reply localization/ekf_localizer/src/ekf_localizer.cpp:655]: fixed https://github.com/autowarefoundation/autoware.universe/pull/1917/commits/6e728a50f0e5f5c98444345f70573f6048c6c80b

**Review by YamatoAndo — APPROVED** [review]

**Pull request merged**